packages/kit/src/vite/utils.js

```diff
@@ -213,11 +217,21 @@ export function get_env(mode, prefix) {
  * @param {(id: string) => import('rollup').ModuleInfo | null} node_getter
  * @param {import('rollup').ModuleInfo} node
  * @param {Set<string>} illegal_imports Illegal module IDs -- be sure to call vite.normalizePath!
- * @param {string} out_dir The directory specified by config.kit.outDir
  */
-export function prevent_illegal_rollup_imports(node_getter, node, illegal_imports, out_dir) {
+export function prevent_illegal_rollup_imports(node_getter, node, illegal_imports) {
 	const chain = find_illegal_rollup_imports(node_getter, node, false, illegal_imports);
-	if (chain) throw new Error(format_illegal_import_chain(chain, out_dir));
+	if (chain) throw new Error(format_illegal_import_chain(chain));
+}
+
+const queryRE = /\?.*$/s;
```

Comment:
We tend to use `pattern` in the codebase rather than `RE`

```suggestion
const query_pattern = /\?.*$/s;
```